packages/collector/src/announceCycle/unannounced.js

```diff
@@ -77,7 +77,13 @@ function tryToAnnounce(ctx, retryDelay = initialRetryDelay) {
   }
   agentConnection.announceNodeCollector((err, rawResponse) => {
     if (err) {
-      logger.debug('Announce attempt failed: %s. Will retry in %s ms', err.message, retryDelay);
+      logger.info(
```

Comment:
I reverted this back to info as discussed. The additional explanation should hopefully make it a bit clearer that this does not need to be reported.